state_version.go

```diff
@@ -243,6 +273,46 @@ func (s *stateVersions) Create(ctx context.Context, workspaceID string, options
 	return sv, nil
 }
 
+func (s *stateVersions) putURL(ctx context.Context, putURL string, data []byte) error {
+	reader := bytes.NewReader(data)
+	req, err := s.client.NewRequest("PUT", putURL, reader)
+	if err != nil {
+		return err
+	}
+
+	return req.Do(ctx, nil)
+}
+
+func (s *stateVersions) Upload(ctx context.Context, workspaceID string, options StateVersionUploadOptions) (*StateVersion, error) {
```

Comment:
💅 add a comment here as well for docs for LSP. the LSP client will list the code docs defined here, not the interface.